src/difference.js

```diff
@@ -0,0 +1,19 @@
+function difference(a, b) {
+  var hash = {}
+  var diff = {}
+
+  b.forEach(function createHash(item) {
+    hash[item] = true
+  })
+
+  a.forEach(function findDiff(item) {
+    if (!hash[item] && !diff[item]) {
+      diff[item] = true
+    }
+  })
+
+  return Object.keys(diff)
```

Comment:
Aaaand now I realized it's of course `Array.prototype.forEach`. My bad — I was caught by my suspicion, `Object.keys` wouldn't have been available in node `0.1x`.

So, I'm good with these lines 😉